content/docs/command-reference/repro.md

```diff
@@ -154,6 +155,9 @@ up-to-date and only execute the final stage.
   corresponding pipelines, including the target stages themselves. This option
   has no effect if `targets` are not provided.
 
+- `--pull` - try automatically pulling cached outputs if they are not present in
```

Comment:
move it down to the right place (after `--downstream`) please

Reply:
@shcheklein Not sure I follow. It is already after/under `--downstream`

Reply:
ah, sorry, GH squeezed text in a confusing way. Please disregard this!